src/main/java/core/basesyntax/ArrayListIndexOutOfBoundsException.java

```diff
@@ -5,3 +5,4 @@ public ArrayListIndexOutOfBoundsException(String message) {
         super(message);
     }
 }
+
```

Comment:
when you commit anything, always review the files, to not commit something inadvertently, like you did here